src/components/PasswordDialog.vue

```diff
@@ -44,6 +44,7 @@
 
 			<NcButton type="primary"
 				class="dialog__button"
+				:disabled="password === null"
```

Comment:
`password === null` doesn't work here, because when the password is not entered, it is an empty string `''`. not `null`

```suggestion
				:disabled="!password"
```